script.js

```diff
@@ -12,4 +29,286 @@ function closePopUp(){
   pop_up_menu.style.display="none";
  }
 
-//popup window will go here for project details
\ No newline at end of file
+
+
+
+
+//workinfo object 
+const workInfo={
+  myRecentPr: 'My Recent Projects',
+  seeProject: 'See Project',
+  bgImg: './images/Img Placeholder (1).png',
+  prTitle1:'Profesional Art Printing Data More',
+  prTitle2:'Multi-Post Stories',
+  prDesc: 'A daily selection of privately personalized reads; no accounts or sign-ups required. This has  been the industry standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+  techList: ['HTML','CSS','Javascript']
+};
+
+
+// markup work section from the wworkinfo array object
+var mainElmnt=document.querySelector('main');
+const divSection=document.createElement('div');
+  mainElmnt.insertBefore(divSection,aboutMeSection);
+  divSection.innerHTML=`<div class="myworks" id="my-works" class="blur">
+        <div class="work-heading">
+            <h4>${workInfo.myRecentPr}</h4>
+            <hr class="hline">
+        </div>
+        <div class="work-details" id="work-details">
+            <div class="work-details-row1">
+                <img src="${workInfo.bgImg}">
+                <div class="details-right1">
+                    <h4>${workInfo.prTitle2}</h4>
+                    <p>${workInfo.prDesc}</p>
+                    <ul>
+                        <li>${workInfo.techList[0]}</li>
+                        <li>${workInfo.techList[1]}</li>
+                        <li>${workInfo.techList[2]}</li>
+                    </ul>
+                    <button class="btn" type="button" id="seeProject-1">${workInfo.seeProject}</button>
+
+                </div>
+            </div>
+            <div class="card-work">
+                    <div class=" work wdr21">
+                    <h4>${workInfo.prTitle1}</h4>
+                    <p>${workInfo.prDesc}</p>
+                    <ul>
+                        <li>${workInfo.techList[0]}</li>
+                        <li>${workInfo.techList[1]}</li>
+                        <li>${workInfo.techList[2]}</li>
```

Comment:
- [ ] I suggest you add the details of each work as objects in an array and loop through the array to create each card of the work section. THis will make your code follow [DRY princiiple](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).